networkx/generators/geometric.py

```diff
@@ -844,3 +845,200 @@ def thresholded_random_geometric_graph(
     )
     G.add_edges_from(edges)
     return G
+
+
+@py_random_state(5)
```

Comment:
New decorator for dispatching to bacend packages:
```suggestion
@py_random_state(5)
@nx._dispatch(graphs=None)
```